flexget/plugins/cli/series.py

```diff
@@ -270,29 +292,29 @@ def register_parser_arguments():
     list_parser = subparsers.add_parser('list', parents=[table_parser],
                                         help='List a summary of the different series being tracked')
     list_parser.add_argument('configured', nargs='?', choices=['configured', 'unconfigured', 'all'],
-                             default='configured',
                              help='Limit list to series that are currently in the config or not (default: %(default)s)')
-    list_parser.add_argument('--premieres', action='store_true',
+    list_parser.add_argument('--premieres', action='store_const', const='premieres',
```

Comment:
That's a good question. I think it was about eliminating the default `False` that `store_true` creates when the flag isn't used? Because for the `--descending`/`--ascending` flags I need to know if they were present to override envs (since both `--ascending` being present or `--descending` not being present result in a value of `False`, so no way to tell which is the case just based on the value). In this case, since there's only one, `False` would mean it's not present, so that would be fine. I can change it back.